autogen/oai/client.py

```diff
@@ -540,6 +514,7 @@ def _completions_create(self, client: OpenAI, params: Dict[str, Any]) -> ChatCom
             # If streaming is not enabled, send a regular chat completion request
             params = params.copy()
             params["stream"] = False
+            params.pop("api_type", None)
```

Comment:
This should be done in `_process_for_azure`

Reply:
I have removed `params.pop()` and added "api_type" to `extra_kwargs`:

```python
class OpenAIWrapper:
    """A wrapper class for openai client."""

    cache_path_root: str = ".cache"
    extra_kwargs = {"cache_seed", "filter_func", "allow_format_str_template", "context", "api_version", "api_type"}
```